api/envoy/extensions/filters/http/rbac/v3/upstream_ip_matcher.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.rbac.v3;
+
+import "envoy/config/core/v3/address.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.rbac.v3";
+option java_outer_classname = "UpstreamIpMatcherProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: RBAC Upstream Ip matcher plugin]
+
+// [#alpha:]
+// [#extension: envoy.filters.http.rbac.matcher.upstream_ip]
```

Comment:
I think you might have a category mismatch.

Reply:
Should be `envoy.filters.http.rbac` ?

Reply:
Above you use `filters.common.rbac`. I would actually move these to common, since I see no reason these couldn't be used for network rbac in the future.